zabbix_metrics_watcher.py

```diff
@@ -231,6 +253,15 @@ def run(zabbix_host, osbs_master, config, instance):
             else:
                 logging.warn("Unhandled status: %r", status)
 
+            spent_in_new = [(now - start).total_seconds() for start in builds_in_new.values()]
+            if spent_in_new:
+                max_spent_in_new = max(spent_in_new)
+                if max_spent_in_new > STUCK_IN_NEW_THRESHOLD_SECS:
+                    logger.info("%s build(s) are in New for, longest: %s sec",
+                                len(builds_in_new), max_spent_in_new)
+                    _send_zabbix_message(zabbix_host, osbs_master,
```

Comment:
We want to avoid sending this message if a build changes the state to New in less than 5 secs - this is acceptable